src/env.ts

```diff
@@ -0,0 +1,112 @@
+import { LogLevel } from '@credo-ts/core'
+import * as dotenv from 'dotenv'
+import * as envalid from 'envalid'
+import { makeValidator } from 'envalid'
+import { singleton } from 'tsyringe'
+
+if (process.env.NODE_ENV === 'test') {
+  dotenv.config({ path: 'tests/test.env.ts' })
+} else {
+  dotenv.config()
+}
+const proofRequestOptions = `{
+    "protocolVersion": "v2",
+    "proofFormats": {
+      "anoncreds": {
+        "name": "drpc-proof-request",
+        "version": "1.0",
+        "requested_attributes": {
+          "companiesHouseNumberExists": {
+            "name": "companiesHouseNumber"
+          }
+        }
+      }
+    }
+  }`
+
+const stringArray = makeValidator((input: string | string[]) => {
+  if (Array.isArray(input)) {
+    return input
+  }
+  if (typeof input !== 'string') {
+    throw new Error('Invalid input type, expected a string')
+  }
+  try {
+    return input.split(',').map((s) => s.trim())
+  } catch (err) {
+    throw new Error('Invalid input for string array')
+  }
+})
+
+const envConfig = {
+  LABEL: envalid.str({ default: 'AFJ Rest', devDefault: 'AFJ Rest Agent' }),
```

Comment:
This should be replaced with veritable now?

Also what is this used for?

Reply:
It appears in all the back and forth didcom messages and is important

Reply:
updated the docs